mycroft/skills/phillips_hue/vocab/en-us/IncreaseColorTemperatureKeyword.voc

```diff
@@ -0,0 +1,10 @@
+cool my lights
+cool the lights
+increase the color temperature of my lights
+increase the color temperature of the lights
+increase the temperature of my lights
+increase the temperature of the lights
+turn up the color temperature of my lights
+turn up the color temperature of the lights
+turn up the temperature of my lights
+turn up the temperature of the lights
```

Comment:
Similar what I suggested before, use a `LightKeyword` and `IncreaseColorTemperatureKeyword` here to clean up the vocab. The [Volume skill](https://github.com/MycroftAI/mycroft-core/tree/master/mycroft/skills/volume) has good examples.